src/app/pages/RuntimeAccountDetailsPage/hook.ts

```diff
@@ -34,19 +33,13 @@ export const useAccountTransactions = (scope: SearchScope, address: string) => {
     // We should use useGetConsensusTransactions()
   }
 
-  const oasisAddress = getOasisAddressOrNull(address)
   const query = useGetRuntimeTransactions(
     network,
     layer, // This is OK since consensus has been handled separately
     {
       limit,
       offset: offset,
-      rel: oasisAddress!,
-    },
-    {
-      query: {
-        enabled: !!oasisAddress,
-      },
```

Comment:
@lubej @csillag are checks like this still needed to handle some empty state

Reply:
oasisAddress used to be async. Since it is sync now and address is always true (loader would fail earlier if not) then I think it is not used to handle empty spaces.